packages/styletron-server/src/cache-to-stylesheets-old-ie.js

```diff
@@ -20,7 +20,9 @@ function cacheToStylesheetsOldIE(cacheObj) {
       mediaSheets = getMediaSheets(cacheObj[key]);
       continue;
     }
-    ruleCount += Object.keys(cacheObj[key]).length;
+    if (typeof cacheObj[key] === 'string') {
```

Comment:
I realized this count excludes pseudos. It needs to count keys if `cacheObj` is an object.